script/monitor-external/capture-dnscap.sh

```diff
@@ -191,6 +194,13 @@ else
     echo "Not capturing IP fragments"
 fi
 
+# Exclude Patterns
+if [ ! -z ${NO_PAT} ]; then
```

Comment:
thank you for asking.

I noted that on my auth server dnscap captures packets sent to the recursive nameservers listed in resolv.conf, for example when trying to send data to data.dnsv6lab.net or whenever I perform update tasks on that FreeBSD box.
I may be wrong, but I think those packets have nothing to do with a yeti-dns auth server role.